catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcDatabaseOperations.java

```diff
@@ -77,6 +77,18 @@ public List<String> listDatabases() {
     }
   }
 
+  /**
+   * Load the schema with the given name.
+   *
+   * @param connection The connection to the database.
+   * @param schemaName The name of the schema.
+   * @return The result set of the schema. Some database may not support fuzzy query and the result
+   *     set may be empty or is not the schema with the given name.
+   * @throws SQLException
+   */
+  protected abstract ResultSet getSchema(Connection connection, String schemaName)
```

Comment:
what could we get from ResultSet besides whether it exists or not?  If we only get exists information I suggest to add `schemaExists` interface

Reply:
Let me think about it.

Reply:
Currently, only `loadSchema` needs to check the existence of a schema, I would not suggest adding a new interface.